ticketmaster.js

```diff
@@ -59,15 +59,48 @@ async function getEventsByArtist(artist) {
   
   // use attraction ids to search to specific events
   response = await fetch(`${eventSearchUrl}&attractionId=${attractionIDs}`)
-  result = await response.json()
-  const events = result._embedded.events
+  result = await response.json();
+  //searching for certain artist throws a TypeError , whereas some will return successfully
+  try{
+    const events = result._embedded.events
+    console.log(events);
+    const eventsData = events.map( ({name, priceRanges, url, images, dates}) => {
+      return { name, priceRanges, url, images, date: dates.start.localDate}
+    })
+  
+    console.log(eventsData)
+    return eventsData
+  }catch(e){
+    if(e instanceof TypeError ){
+      console.log(e);
+    }
+  }
 
-  const eventsData = events.map( ({name, priceRanges, url, images, dates}) => {
-    return { name, priceRanges, url, images, date: dates.start.localDate}
-  })
 
-  console.log(eventsData)
-  return eventsData
 }
 
 // TODO: combine all of these into a single search function
+
+
+$(document).ready(function(){
```

Comment:
I don't think we need to wrap this in a $(document). ready function. If the user is submitting a search, the form should already be loaded